daemon/network.go

```diff
@@ -1005,49 +1005,44 @@ func getPortMapInfo(sb *libnetwork.Sandbox) nat.PortMap {
 	}
 
 	for _, ep := range sb.Endpoints() {
-		pm, _ = getEndpointPortMapInfo(ep)
-		if len(pm) > 0 {
-			break
-		}
+		ep.Network()
```

Comment:
```suggestion
```

`Network` seems to just return a value without mutating the `ep` - I think this is unused and can be removed?